apps/etterlatte-behandling/.run/etterlatte-behandling.dev-gcp.run.xml

```diff
@@ -55,6 +55,8 @@
       <env name="JOBB_DOEDSMELDINGER_REMINDER_OPENING_HOURS" value="00-23" />
       <env name="JOBB_SAKSBEHANDLER_OPENING_HOURS" value="00-23" />
       <env name="JOBB_METRIKKER_OPENING_HOURS" value="00-23" />
+      <env name="KODEVERK_CLIENT_ID" value="dev-gcp.team-rocket.kodeverk-api" />
+      <env name="KODEVERK_URL" value="https://kodeverk-api.nav.no/api/v1/kodeverk" />
     </envs>
```

Comment:
Burde dette også legges til i `.nais` for dev og prod?

Reply:
Absolutt! Noe som ble borte her gitt

Reply:
Ah nei jeg hadde bare glemt å legge det inn her mens for dev og prod la jeg det inn her https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/5555

Reply:
Ah, jeg sjekket ikke filene. Bare såg de manglet fra PR.